src/mixins/crownConfig.js

```diff
@@ -50,10 +60,10 @@ export default {
   watch: {
     highlighted(highlighted) {
       if (highlighted) {
-        this.shapeView.highlight();
+        this.shapeView.highlight(this.selectShape, { highlighter: defaultHighlighter });
         this.addCrown();
       } else {
```

Comment:
Tiny style thing: how would you feel about using an early return and not an else here?